Assets/Scripts/Animator/FaceLandmarksRecognizer.cs

```diff
@@ -53,7 +55,11 @@ public class FaceLandmarksRecognizer {
     /// <summary>The translation vector for SolvePnP.</summary>
     private float[] _translationVector = new float[3];
     /// <summary>Exported native C function for SolvePnP.</summary>
+#if UNITY_MAC_INTER
```

Comment:
I prefer UNITY_MAC_X64 and UNITY_MAC_ARM64 for such variables

(Strictly speaking, Intel is not a CPU architecture, it's a brand. Binary code for Intel X64 CPU can run on AMD X64 also.)